### PR TITLE
Adds routes to technical.md

### PR DESCRIPTION
**Pull request by mk4111**
Moves information on #378 to technical.md; adds endpoints, methods and auth

### CODE REVIEW

File: technical.md [hunk 1]
```diff
@@ -4,10 +4,10 @@
 All of our data is managed by modules, which we have published as plugins. Below, we will list out the data and their types, contained per table, per plugin.
 
 ### List of Plugins:
-* [tags-system](###tags-system)
-* [pg-people](###pg-people)
-* [pg-challenges](###pg-challenges)
-* [pg-insights](###pg-insights)
+* [tags-system](#tags-system)
```
Comment (JMurphyWeb) [inline technical.md]: Is this a link to the plugins?

Comment (SimonLab) [inline technical.md]: @JMurphyWeb There are links to the title inside the markdown page

File: technical.md [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
+* [pg-people](#pg-people)
+* [pg-challenges](#pg-challenges)
+* [pg-insights](#pg-insights)
 
 ### [tags-system](npmjs.com/package/tags-system)
```
Comment (SimonLab) [inline technical.md]: I think you need to add the full url of the links otherwise it will try to access the link from github I think

Comment (mk4111) [inline technical.md]: Ok!

**Pull request merged**